src/components/ConcentratedLiquidity/PositionCard/index.tsx

```diff
@@ -0,0 +1,55 @@
+import React from 'react';
+import { useTranslation } from 'react-i18next';
+import { Box, DoubleCurrencyLogo, Text } from 'src/components';
+import { BlackBox, BlackBoxContent, Card, Data, Price, Row } from './styles';
+import { PositionCardProps } from './types';
+
+const PositionCard: React.FC<PositionCardProps> = (props) => {
+  const { currency0, currency1 } = props;
+  const { t } = useTranslation();
+
+  return (
+    <Card>
```

Comment:
1. there is horizontal scroll
![Screenshot 2023-03-07 at 8 45 17 AM](https://user-images.githubusercontent.com/6604146/223311433-123d52af-59c7-4458-8022-6027363d707e.png)

2. also please make sure this UI looks good in mobile device
3. any idea why colors are not proper in storybook ? please make sure it looks good in white mode.